lib/process-release.js

```diff
@@ -0,0 +1,99 @@
+var semver = require('semver')
+var url = require('url')
+var path = require('path')
+
+var bitsre = /(x86|x64)/
+
+// Captures all the logic required to determine download URLs, local directory and 
+// file names and whether we need to use SHA-1 or 2. Inputs come from command-line
```

Comment:
s/2/SHA-256/?  SHA-2 is a family of hash functions, of which SHA-256 is just one.